client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -170,7 +174,7 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 				{ flowSteps.map( ( step ) => (
 					<Route
 						key={ step.slug }
-						path={ `/${ flow.variantSlug ?? flow.name }/${ step.slug }` }
+						path={ `/${ flow.variantSlug ?? flow.name }/${ step.slug }/:lang?` }
```

Comment:
I have a concern: this feels like it will no longer match `/:flow/:step` without the trailing `/`, as I am not sure if the optionality applies to the `/`.

Reply:
> I am not sure if the optionality applies to the /.
Yes, it is applicable, regular /:flow/:step are working properly.